src/scripts/main.js

```diff
@@ -6,3 +6,17 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard tbody');
```

Usually, tables should have 'thead', 'tbody' and 'tfoot' elements for semantical purposes and styling. So you should add rows inside `<body>`, not directly inside the `<table>`. As you may see, your table is kind of different from that is in the task description.